node-graph/graph-craft/src/document/value.rs

```diff
@@ -194,6 +202,7 @@ impl<'a> TaggedValue {
 			TaggedValue::OptionalColor(_) => concrete!(Option<graphene_core::Color>),
 			TaggedValue::ManipulatorGroupIds(_) => concrete!(Vec<graphene_core::uuid::ManipulatorGroupId>),
 			TaggedValue::VecDVec2(_) => concrete!(Vec<DVec2>),
+			TaggedValue::Segments(_) => concrete!(graphene_core::raster::IndexNode<u32>),
```

Comment:
> I think this should be `Vec<ImageFrame>`

Hmm, are you sure? Because I get crashes if I modify it to `Vec<ImageFrame>`:
```
dex_bg.js:1163 error	Failed to update executor:No implementations found for graphene_std::memo::EndLetNode<_> with input: Concrete(&graphene_core::raster::image::ImageFrame) and parameters: [(Concrete(()) -> Concrete(alloc::vec::Vec<graphene_core::raster::image::ImageFrame>))].Other Implementations found: {NodeIOTypes { input: Generic(T), output: Concrete(graphene_core::raster::image::ImageFrame), parameters: [(Concrete(()) -> Concrete(graphene_core::raster::image::ImageFrame))] }: 0x9fb, NodeIOTypes { input: Generic(T), output: Concrete(graphene_core::raster::image::ImageFrame), parameters: [(Concrete(()) -> Concrete(graphene_core::vector::vector_data::VectorData))] }: 0x9fc}
```

Reply:
This error complains that you try to return a vec of image frames instead of a single image frame, did you forget the index node?

Reply:
> This error complains that you try to return a vec of image frames instead of a single image frame, did you forget the index node?

You're right, my bad.